### PR TITLE
ci: validate user before creating tag

### PR DESCRIPTION
**Pull request by subhamkrai**
we need to validate create tag action
so that only authorized users can trigger
that action.

[skip-ci]

Signed-off-by: subhamkrai <srai@redhat.com>

**Description of your changes:**

**Which issue is resolved by this Pull Request:**
Resolves #

**Checklist:**

- [ ] **Commit Message Formatting**: Commit titles and messages follow guidelines in the [developer guide](https://rook.io/docs/rook/master/development-flow.html#commit-structure).
- [ ] **Skip Tests for Docs**: Add the flag for skipping the build if this is only a documentation change. See [here](https://github.com/rook/rook/blob/master/INSTALL.md#skip-ci) for the flag.
- [ ] **Skip Unrelated Tests**: Add a flag to run tests for a specific storage provider. See [test options](https://github.com/rook/rook/blob/master/INSTALL.md#test-storage-provider).
- [ ] Reviewed the developer guide on [Submitting a Pull Request](https://rook.io/docs/rook/master/development-flow.html#submitting-a-pull-request)
- [ ] Documentation has been updated, if necessary.
- [ ] Unit tests have been added, if necessary.
- [ ] Integration tests have been added, if necessary.
- [ ] Pending release notes updated with breaking and/or notable changes, if necessary.
- [ ] Upgrade from previous release is tested and upgrade user guide is updated, if necessary.
- [ ] Code generation (`make codegen`) has been run to update object specifications, if necessary.

### CODE REVIEW

File: .github/workflows/create-tag.yaml
Comment (subhamkrai) [inline .github/workflows/create-tag.yaml:17]: @travisn  you can delete `RELEASE_ADMINS` from the repo secrets

Comment (subhamkrai) [reply .github/workflows/create-tag.yaml:17]: also, we need to backport this to 1.7 and 1.6 right?

Comment (travisn) [reply .github/workflows/create-tag.yaml:17]: Ok `RELEASE_ADMINS` is removed

File: .github/workflows/create-tag.yaml
Comment (travisn) [inline .github/workflows/create-tag.yaml:17]: Is it possible to test a similar condition on your fork before we merge this? Perhaps on a different action, just to make sure the syntax is correct.
```suggestion
    if: github.repository == 'rook/rook' && contains('travisn,leseb,BlaineEXE', github.actor)
```

Comment (travisn) [reply .github/workflows/create-tag.yaml:17]: Let's also make this list the same as maintainers, so please add `jbw976,galexrt,satoru-takeuchi`

Comment (subhamkrai) [reply .github/workflows/create-tag.yaml:17]: can you try trigger this action https://github.com/subhamkrai/rook/actions/workflows/create-tag.yaml here

**Review by travisn — APPROVED** [review]

**Comment by travisn** [comment]
@subhamkrai This needs manual backporting

**Pull request merged**
